src/test/java/io/xstefank/wildfly/bot/PRRuleTitleBodyCheckTest.java

```diff
@@ -122,4 +124,77 @@ void testFailedTitleBodyCheck() throws IOException {
                     Util.verifyFormatFailure(repo, gitHubJson, "title");
                 });
     }
+
+    @Test
+    void testTitleBodyCheckTitleRegex() throws IOException {
+        GitHubJson gitHubJson = GitHubJson.builder(VALID_PR_TEMPLATE_JSON)
+                .title("[WFLY-00000] Metrics PR")
+                .action(Action.OPENED)
```

Comment:
This line shouldn't be needed in any of the tests. The default action is "opened"